spec/src/snapshot/index.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$id": "https://datareplication.io/doc/jsonschema/snapshot/index.schema.json",
+    "$schema": "https://json-schema.org/draft-07/schema",
```

Comment:
It's valid, but can you update to the latest json-schema? 
[2020-12](https://json-schema.org/specification-links.html#2020-12)

Reply:
I've changed the schema version accorrdingly.